docs/release-strategy.md

```diff
@@ -68,3 +68,22 @@ The following are the steps for how Y-stream and Z-stream releases gets cut.
 1. Announce release via the following:
     - The `#announce` channel on Slack
     - The `announce` mailing list
+
+## Release notes
+
+The project maintains a single `CHANGELOG.md` file that documents all releases.
+To ensure our users are well-informed about new features, improvements, and breaking changes, we
+maintain a `CHANGELOG.md` file. This file serves as a centralized place to document changes
+that will be included in the next major (X) release.
```

Comment:
```suggestion
that will be included in the next major (X) release.
```
Y-streams as well?

Reply:
The Y-streams are backports only right? So they will only include critical bug fixes, no features nor breaking changes. From my perspective, it's just Major release updates.

Reply:
Z-Streams are backports only

Reply:
Rah sorry, actually I mixed things up. I was talking about (Y) releases all along and not (X). In the end, both X and Y will get the same treatment. Thanks for pointing this out!

Reply:
Np at all!